openapi-templates/api_client.mustache

```diff
@@ -1,6 +1,6 @@
 {{>partial_header}}
 
-import json
+import orjson
```

Comment:
If you introduce it into the generated openapi client, you'll need to add it to the `setup.py` (not just requirements.txt, as that is only for the server).